From: "Barak A. Pearlmutter" <barak+git@cs.nuim.ie>
Date: Tue, 9 Aug 2011 14:11:28 +0200
Subject: desktop

Fedora added source file: ettercap.desktop

From Fedora ettercap repository,
git://pkgs.fedoraproject.org/ettercap.git
3f1eb7a83fd3f7c51ce5f62ef3429982116194b8

Remove deprecated boilerplate line from ettercap.desktop

$ desktop-file-validate ettercap.desktop
ettercap.desktop: warning: key "Encoding" in group "Desktop Entry" is deprecated

tweak invocation in desktop/ettercap.desktop
---
 ettercap.desktop |    8 ++++++++
 1 file changed, 8 insertions(+)
 create mode 100644 ettercap.desktop

diff --git a/ettercap.desktop b/ettercap.desktop
new file mode 100644
index 0000000..02ca686
--- /dev/null
+++ b/ettercap.desktop
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Name=Ettercap
+Comment=Network sniffer/analyser
+Exec=su-to-root -X -c "/usr/sbin/ettercap --gtk"
+Icon=ettercap
+Terminal=false
+Type=Application
+Categories=Network;
